backend/tests/datarooms/test_models.py

```
@@ -80,3 +81,32 @@ def test_dataroom_item_order_creation(dataroom, document):
+        role=DataroomCollaborator.ROLE_COLLABORATOR,
+        invited_by=user,
+    )
+    assert isinstance(collab, DataroomCollaborator)
+    assert str(collab) == f"{collab_user.email} - {dataroom.name} (collaborator)"
+    assert collab.dataroom == dataroom
+    assert collab.user == collab_user
+    assert collab.invited_by == user
+    assert collab.role == "collaborator"
+
+    # Test uniqueness
+    with pytest.raises(Exception):
```

Ruff (0.16.2)

[warning] Do not assert blind exception: `Exception`

(B017)